### ISSUE TITLE
Response doesn't contain config.meta since 0.19.0 

### ISSUE THREAD

**Issue by ivan-kalinin** [axios/axios#2252]
On version 0.18.0 could be use meta parameter like this
```js
const api = axios.create({....})
api.get('/someroute', {meta: {somemeta: true}})
```
It was available inside of config on response. But since update on 0.19.0 I've stopped watching meta there. No found any information about this in changelog also. It may be not a bug, but can you explain this missing, please.

**To Reproduce**
```js
var axios = require("axios") // with  0.18.0 and 0.19.0 responses are different
axios.get('https://www.google.com/', {meta: {somemeta: true}}).then(res => {console.log(res)})
```

**Expected behavior**
Expect to see `meta` object inside of `res.config`

**Environment:**
 - Axios Version 0.19.0
 - OS: OS 12.2
 - Additional Library Versions: React 16.8.6, React Native 0.59.9

**Comment by jphilipstevens** [comment]
I believe that there was a breaking change. The fix should be in this PR: https://github.com/axios/axios/pull/2207

for now downgrade to 0.18 is what we were reccomended

**Comment by 93hsc** [comment]
mark

**Comment by syyam** [comment]
> I believe that there was a breaking change. The fix should be in this PR: #2207
> 
> for now downgrade to 0.18 is what we were recommended

How do you downgrade axios?

**Comment by chinesedfan** [comment]
#2207 has been merged and 0.19.1 includes it.
